.git-blame-ignore-revs

```diff
@@ -0,0 +1,5 @@
+# Copyright 2024 The Khronos Group Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Apply initial formatting
+4ceab17e94e73c918130a49ad1b21d5e4c14e7ec
```

Comment:
Isn't this ignoring just this single commit? What about future reformatting that takes place when commit(s) are pushed?

Reply:
Thanks for reminding me, this commit hash needs update after rebases I done.

Ideally no further commits need to be added here, because now the entire repo is formatted and all formatting violations will trip CI, hence no pure formatting-related commits need to be made. Contributors are highly encouraged to configure their development environment to format on save. (eg. VS Code has [`editor.formatOnSave`](https://code.visualstudio.com/docs/editor/codebasics#_formatting) option and multiple C/C++ formatting extensions available, most notably [ms-vscode.cpptools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) or [llvm-vs-code-extensions.vscode-clangd](https://marketplace.visualstudio.com/items?itemName=llvm-vs-code-extensions.vscode-clangd))

Reply:
Please resolve this when you've updated the commit hash.

Reply:
It's been updated with the latest rebase.

Reply:
Sorry. I edited my original comment that said "please resolve" to add the request about documentation which you probably missed. Probably `BUILDING.md` is the best place for the documentation.

Reply:
Documented VS Code. I looked into (Neo)vim, but the phase space is too large.